src/analyses/goto_check_c.cpp

```diff
@@ -2170,27 +2170,6 @@ void goto_check_ct::goto_check(
     }
     else if(i.is_throw())
     {
-      if(
```

Comment:
Shouldn't that go into `java_bytecode/goto_check_java` then?

Reply:
Sort of, except `goto_check_java` should actually just be removed. See #6687 (and then #6686).

Reply:
And then also `goto_check_java` actually has this code. I previously made it conditional on `--pointer-check` in there, just to realise that it actually is dead code.